File: tuts/005-cloudfront-gettingstarted/python/scenario_getting_started.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

import uuid
import boto3
from cloudfront_wrapper import CloudFrontWrapper


def run_scenario():
    client = boto3.client("cloudfront")
    wrapper = CloudFrontWrapper(client)
    print(f"Running CloudFront getting started scenario...")
    # TODO: implement setup, interact, teardown
    print("Scenario complete.")


if __name__ == "__main__":
    run_scenario()

File: tuts/005-cloudfront-gettingstarted/ruby/scenario_getting_started.rb
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

require_relative 'cloudfront_wrapper'

def run_scenario
  client = Aws::CloudFront::Client.new
  wrapper = CloudFrontWrapper.new(client)
  puts "Running CloudFront getting started scenario..."
  # TODO: setup, interact, teardown
  puts "Scenario complete."
end

run_scenario if __FILE__ == $PROGRAM_NAME
